src/components/ModifierPlayground/useTimingPlayground/index.tsx

```diff
@@ -202,38 +221,58 @@ export default function useTimingPlayground() {
       {(easing === "bezier" ||
         (nestedEasing === "bezierFn" && canNestEasing(easing))) && (
         <>
-          <Range
-            label="x1"
-            min={0}
-            max={1}
-            step={0.01}
-            value={x1}
-            onChange={setX1}
-          />
-          <Range
-            label="y1"
-            min={-1}
-            max={2}
-            step={0.01}
-            value={y1}
-            onChange={setY1}
-          />
-          <Range
-            label="x2"
-            min={0}
-            max={1}
-            step={0.01}
-            value={x2}
-            onChange={setX2}
-          />
-          <Range
-            label="y2"
-            min={-1}
-            max={2}
-            step={0.01}
-            value={y2}
-            onChange={setY2}
-          />
+          {!isMobile && (
+            <CollapseButton
+              label="Collapse the controls section"
+              labelCollapsed="Expand the controls section"
+              collapsed={bezierCollapsed}
+              onCollapse={() => setBezierCollapsed((prevState) => !prevState)}
+              className={styles.collapseButton}
+            />
+          )}
+          <Collapsible
+            collapsed={bezierCollapsed}
+            lazy={false}
+            onCollapseTransitionEnd={(newCollapsed) => {
+              setBezierCollapsed(newCollapsed);
+            }}
+            /* As range sliders may hide their overflow on the boundaries during the collapse animation,
+             * we need to shorten the duration of the animation to 1ms (as 0ms will just not show it). */
+            animation={{ duration: 1 }}
```

Comment:
why can't we use just the default one? it kinda looks cool I guess

https://github.com/software-mansion-labs/reanimated-beta-docs/assets/39658211/a7040c6e-397d-4a77-8664-0079c69bf037

Reply:
That's because when you set one of those settings to the most left / most right side (e.g. set it to 0 or 100) the slider thumb is getting cutted - that's because overflow is being set up to `hidden` during the animation. 

I've tried to omit this, but when I've tried to force overflow to be always on the `visible` option - the content of the collapsible box was always visible during the animation. 

The most possible workaround is to manage the padding / width of the box, but in my opinion it won't look good if the whole section will get 1rem horizontal padding (or the padding or width will change during the animation).

So I think we got two options there - we can just treat this problem like it doesn't happen at all or we can disable the animation - what do you think?